cmd/src/campaign_progress_printer.go

```diff
@@ -134,13 +134,30 @@ func (p *campaignProgressPrinter) PrintStatuses(statuses []*campaigns.TaskStatus
 	}
 
 	for _, ts := range newlyCompleted {
+		p.progress.WriteLine(output.Linef("", output.StylePending, "%s", ts.RepoName))
+		lines, err := diffLines(ts)
+		if err != nil {
+			p.progress.Verbosef("%-*s failed to display status: %s", p.maxRepoName, ts.RepoName, err)
+			continue
+		}
+		for _, line := range lines {
+			p.progress.Verbosef("\t%s", line)
+		}
+
 		statusText, err := taskStatusText(ts)
 		if err != nil {
 			p.progress.Verbosef("%-*s failed to display status: %s", p.maxRepoName, ts.RepoName, err)
 			continue
 		}
 
-		p.progress.Verbosef("%-*s %s", p.maxRepoName, ts.RepoName, statusText)
+		verboseDiffStat, err := finalDiffStatLine(ts)
+		if err != nil {
+			p.progress.Verbosef("%-*s failed to display status: %s", p.maxRepoName, ts.RepoName, err)
+			continue
+		}
+
+		p.progress.Verbosef("  %s", verboseDiffStat)
+		p.progress.Verbose("")
```

Comment:
Same question as above: could we gate this on the verbose flag and not do the work if it's disabled?

Reply:
Done!